src/curated-corpus/pages/SchedulePage/SchedulePage.tsx

```diff
@@ -438,16 +451,18 @@ export const SchedulePage: React.FC = (): JSX.Element => {
                   key={data.scheduledDate}
                 >
                   <Grid item xs={12}>
-                    <Grid container justifyContent="center">
-                      <Grid item>
-                        <Typography className={classes.heading} variant="h2">
-                          {getDayAndSyndicatedCountHeading(data)}
-                        </Typography>
-                      </Grid>
-                      <Grid item xs={12}>
-                        <hr />
-                      </Grid>
-                    </Grid>
+                    <Box mt={3}>
+                      <Accordion>
+                        <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                          <Typography className={classes.heading}>
+                            {getDayAndSyndicatedCountHeading(data)}
+                          </Typography>
+                        </AccordionSummary>
+                        <AccordionDetails>
+                          <ScheduleSummaryLayout scheduledItems={data.items} />
+                        </AccordionDetails>
+                      </Accordion>
+                    </Box>
```

Comment:
Thoughts on making the `<Accordion>` jsx/html part of `<ScheduleSummaryLayout>` jsx? That way we'd have a cleaner jsx here i.e:

```
<Grid item xs={12}>
   <ScheduleSummaryLayout>
</Grid>
```

This is a good option if we plan to always keep this component an accordion component and don't plan to re-use it somewhere else in some other UI format

Reply:
I'd have to send the entire query response to it in that case - with the component as is I'm keeping the scope of it smaller and it's easier to unit test with just ScheduledCorpusItem[] sent through.